src/AssetForm/containers/draft.js

```diff
@@ -0,0 +1,146 @@
+/**
+ * Utility components and functions for dealing with the current draft.
```

Comment:
This file has some old cruft in it. Only look at the withDraft() function. Other functions need removing.